modules/@apostrophecms/doc-type/index.js

```diff
@@ -0,0 +1,26 @@
+module.exports = {
```

Comment:
Placing this file in `modules/@apostrophecms/doc-type/index.js` causes Apostrophe to load it as additional configuration for `@apostrophecms/doc-type`, the base class of all modules.